Update base OS info #268 (#298)
rockstor/rockstor-doc: installation/install.rst

-------------------------

There is nothing about Rockstor that requires special hardware. It's Linux, and
specifically Rockstor 4 is "Built on openSUSE" while v3 used CentOS, so it can

> be installed on a wide range of commodity hardware. With 4 gaining ARM64

Reviewer: ```suggestion
be installed on a wide range of commodity hardware, with 4 gaining ARM64
```

2. Yet another recommendation used by some community members is HP Proliant
   Microserver Gen8.

3. Raspberry Pi4 and RPi 400 have also both been reported as working as

>    intended via our newer Rockstor 4 "Build on openSUSE" variant. N.B. RPi 400

Reviewer: ```suggestion
   intended via our newer Rockstor 4 "Built on openSUSE" variant. *N.B.* RPi 400
```

   requires at least a 15.3 profile for the internal keyboard to work.

Upgrading Rockstor
------------------
Rockstor is under continuous development and we generally release updates in
small batches. These updates are easy to install. But we do roll-out major

> releases that require a complete re-installs. Upgrading from Rockstor 3 (CentOS

Reviewer: ```suggestion
releases that require a complete re-install. Upgrading from Rockstor 3 (CentOS
```

based) to Rockstor 4 "Built on openSUSE" is one such update.
But such updates are very rare.

Non re-install Rockstor updates can be installed in two ways :

1. Install updates from the Web-UI (recommended):

> On the Rockstor Web-UI, far top-right, you will see an upward facing arrow if

Reviewer: This seems like a good spot for a screenshot focused on the "upward-facing arrow".

Author: Agreed, I'll add via another commit.

any Rockstor package updates are available. All upstream packages except the
'rockstor' package can similarly be installed by clicking on a flashing

> wifi-like icon. Again this is not shown if no updates are available. But it

Reviewer: This also seems like a good spot for a screenshot of the "wifi-like" icon.
Additional minor edit suggestion:
```suggestion
wifi-like icon. Again, this is not shown if no update is available, but it
```

Author: OK. Again I'll add via a commit as merging here might be tricky given I've now also added a pic in here via other changes. I may be in for a merge conflict with all these in GitHub and outside of GitHub changes on the same file. Oh well.

does show, as per the up-arrow, if there is a rockstor package update
available.
But only the up-arrow will actually update the main rockstor package.

> These disparate but related mechanisms allows folks to choose to only update

Reviewer: ```suggestion
These disparate but related mechanisms allow users to choose to only update what they want:
```

Author: OK, this was likely a little too colloquial. Cheers.

> all packages bar the rockstor package (wifi-like icon).

Reviewer: Suggestion below (pardon the potential lack of formatting for the moment):
```suggestion
- Wifi-like icon: all packages bar the rockstor package.
- Up arrow: only the rockstor package.
```

Author: Skipping this change as I need to double check all this actually.

Reviewer: Yeah, I hesitated on that one but that's how I think I remembered it and your edits comforted what I remembered... I should have been more careful and verified myself first.

See the following section for details on upgrading the Rockstor package plus
all pending upstream updates :ref:`update_channels`.

> 2. Alternatively, for advanced users only, one can updated from the Command

Reviewer: ```suggestion
2. Alternatively, for advanced users only, one can update from the Command
```
